api/src/main/java/org/apache/any23/vocab/Vocabulary.java

```diff
@@ -142,12 +142,12 @@ public IRI getProperty(String name, IRI defaultValue) {
      */
     public IRI getPropertyCamelCase(String property) {
         String[] names = property.split("\\W");
-        String camelCase = names[0];
+        StringBuffer camelCase = new StringBuffer().append(names[0]);
```

Comment:
`StringBuilder` would be a better choice here; it doesn't need synchronization.

`StringBuilder camelCase = new StringBuilder(property.length()).append(names[0]);